core/commands/config.go

```diff
@@ -502,7 +503,13 @@ func editConfig(filename string) error {
 		return errors.New("ENV variable $EDITOR not set")
 	}
 
-	cmd := exec.Command("sh", "-c", editor+" "+filename)
+	var cmd *exec.Cmd
+	switch runtime.GOOS {
+	case "windows":
+		cmd = exec.Command("cmd", "/C", editor, filename)
+	default:
+		cmd = exec.Command("sh", "-c", editor+" "+filename)
+	}
```

Comment:
Sorry to ask you again but actually all of this is an issue because we are determined to use the shell.
Which we dont need. we just need to handle the path resolution here.
```suggestion
	path, err := exec.LookPath(editor)
	if err != nil {
		return err
	}
	cmd := exec.Command(editor, filename)
```
Does that work on windows ?

Reply:
Edit:
Actually `exec.Command` can do this for us so this can be as easy as:
```suggestion
	cmd := exec.Command(editor, filename)
```
Am I missing something here ?

Reply:
> Does that work on windows?

Thx, it works. Updated.